ocaml/xapi/message_forwarding.ml

```diff
@@ -745,8 +745,22 @@ functor
         info "Pool.eject: pool = '%s'; host = '%s'"
           (current_pool_uuid ~__context)
           (host_uuid ~__context host) ;
+        let master = Helpers.get_master ~__context in
         let local_fn = Local.Pool.eject ~host in
+        let other =
+          Xapi_pool_helpers.get_master_slaves_list ~__context
```

Comment:
If the master host is removed from the list, why use this call instead of `Db.Host.get_all ~__context`?

Reply:
Good point - will use it instead